CHANGES.rst

```diff
@@ -5,6 +5,9 @@ Version 2.2.1
 
 Unreleased
 
+-   Fix router so that ``/path/`` will match a rule ``/path`` if the
+    rule has strict-slashes == False. :issue:`2467`
```

Comment:
> rule set `strict_slashes=False`.

Reply:
Could also be set by the map adapter though

Reply:
"if strict slashes mode is disabled for the rule"

Reply:
My main point was that it wasn't code-formatted, and that `strict-slashes` had a dash instead of underscore.